AddonCatalog.json

```diff
@@ -93,6 +93,14 @@
       "zip_url": "https://github.com/hamish2014/FreeCAD_assembly2/archive/refs/heads/master.zip"
     }
   ],
+  "Assembly2MuJoCo": [
+    {
+      "repository": "https://github.com/AnesBenmerzoug/FreeCAD-Assembly2MuJoCo",
+      "git_ref": "v0.2.0",
+      "branch_display_name": "main",
```

Comment:
I'd suggest not displaying the branch as "main" if you are going to hard-pin to a specific tag. Also, the Addon Manager will never offer an update to this addon when set up like this, and every time you  release a new version you will need to come to this repo and make a new entry for your new tag. Your users will then need to actually choose a new tag from a drop-down of available tags if they want to update. If that's the update behavior you want that is fine, but I find that most addon authors don't want to wait for a PR here to push an update to their addon's users.